rest/data/project.go

```diff
@@ -38,8 +39,8 @@ func (pc *DBProjectConnector) FindProjectById(id string) (*model.ProjectRef, err
 }
 
 // CreateProject inserts the given model.ProjectRef.
-func (pc *DBProjectConnector) CreateProject(projectRef *model.ProjectRef) error {
-	err := projectRef.Insert()
+func (pc *DBProjectConnector) CreateProject(projectRef *model.ProjectRef, u *user.DBUser) error {
+	err := projectRef.Add(u)
```

Comment:
It's a little odd to me that a function with the signature `*ProjectRef.Add(*user.DbUser)` also inserts the *ProjectRef. Should these be separate functions?

Reply:
I don't think so - this combines insert with updating permissions. I feel like calling it Add implies that, but I can rename it to something like CreateProject

Reply:
No need, SGTM.